docs/docs/recipes/transforming-data.md

```diff
@@ -54,3 +54,39 @@ export const query = graphql`
 
 - [Tutorial on transforming Markdown to HTML](/tutorial/part-six/#transformer-plugins) using `gatsby-transformer-remark`
 - Browse available transformer plugins in the [Gatsby plugin library](/plugins/?=transformer)
+
+## Transforming images into grayscale using GraphQL
+
+`gatsby-image` is a React component designed to work seamlessly with Gatsby’s [native image processing](https://image-processing.gatsbyjs.org/) capabilities powered by GraphQL and [gatsby-plugin-sharp] `/packages/gatsby-plugin-sharp/` to easily and completely optimize image loading for your sites.
+
+### Prerequisites
+
+- A Gatsby site with `gatsby-config.js` and an `index.js` page
+- The `gatsby-image`, `gatsby-transformer-sharp`, and `gatsby-plugin-sharp` packages installed and added to the plugins array in `gatsby-config`
```

Comment:
```suggestion
- The `gatsby-image`, `gatsby-transformer-sharp`, and `gatsby-plugin-sharp` packages installed and added to the plugins array in `gatsby-config`
```
Should this be a pre-req? Or is it worth including the config example?

Reply:
Compare it to the other image recipes: they should be consistent with one another.

Reply:
This is consistent with other image recipes